### DISCUSSION TITLE
[FIXED] Snowisuke

### DISCUSSION THREAD

**Discussion by bigjackson** [nzp-team/nzportable#763] [Map Releases]
**Welcome to Not Lithuania Mountain**, where Snowisuke takes place.

snowisuke is a map i've been making for a while, honestly just because of lack of ideas, i feel like there's too many trees.
 
skybox: striga from the NZP discord (i'll change it to a github tag if they want me to lol)

> _**QUAKESLIDE TOO HARD GOTTA MAKE REAL MAPS NOW**_

### [DOWNLOAD HERE](https://github.com/bigjackson/nzpmapsrepo/raw/main/snowisuke.zip)

**_screenshots:_**

![image](https://github.com/nzp-team/nzportable/assets/60328163/8a16849f-c63e-4ccd-b512-27f4f4d9e5b3)
![image](https://github.com/nzp-team/nzportable/assets/60328163/de86f746-35a6-40ba-a9c3-3db93975101c)
![image](https://github.com/nzp-team/nzportable/assets/60328163/ab6c57e5-7e43-43d2-9c80-2a5da0711a7f)

**Comment by bigjackson** [comment]
fixed the issue with the skybox, i forgot to package it in.

**Reply by bigjackson** [reply]
i fixed most of the problems with the map, i don't think the third window in the map works but i could care less now to be honest.

**Reply by bigjackson** [reply]
potentially fixed a problem with zombies spawning outside of the door

**Reply by bigjackson** [reply]
**THE MAP IS FINALLY AT A PLAYABLE STATE**

basically i just forgot to activate a zombie spawn, and i left it like that for an entire month

**Reply by ToonLink0001** [reply]
nice

**Comment by Jeyphr** [comment]
Check out my review (when it comes out) : https://youtu.be/gAn1P9qrOd0

**Reply by Jeyphr** [reply]
It's out now!
